sdk/core/Azure.Core/src/PolicyLocation.cs

```diff
@@ -3,9 +3,18 @@
 
 namespace Azure.Core
 {
+    /// <summary>
+    /// Represents a position of the policy in the pipeline.
```

Comment:
nit:  The filename and class name don't match.

Reply:
I thought we had stylecop rule for this.

Reply:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/2223

Reply:
Pretty sure that we do.   I've been nagged by it a couple of times this week.